docker-compose/local/django/celery/beat/start

```diff
@@ -5,4 +5,4 @@ set -o nounset
 
 
 rm -f './celerybeat.pid'
-celery -A config.celery_app beat -l INFO -Q celery,analyzer
+celery -A config.celery_app beat -l INFO
```

Comment:
This is not necessary on the beat call, only the worker call. We aren't doing this in prod and it was preventing beat from running in dev.

Reply:
Odd. I thought it would only listen for 1 by default.